EnglishFilter/sections/content_blocker.txt

```diff
@@ -1314,6 +1314,10 @@ topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru
 freemc.host#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72650#issuecomment-771199740
 @@||googletagmanager.com/gtag/js$domain=exe.app
+! anti-adb check-plagiarism.com
+@@||check-plagiarism.com^$generichide
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=check-plagiarism.com
+@@||pagead2.googlesyndication.com/pagead/js/r*/r*/show_ads_impl.js$domain=check-plagiarism.com
```

Comment:
better like in all other rules:
`@@||pagead2.googlesyndication.com/pagead/js/r*/show_ads_impl.js$***`
or
`@@||pagead2.googlesyndication.com/pagead/js/*/show_ads_impl.js$***`

Reply:
Sure.

Reply:
The internet is full of anti-adblocks now. Even normal browsing has anti-adblocks like these sites. I went to all these sites to paraphrase, suddenly anti-adblocks came.